libs/datasets/timeseries.py

```diff
@@ -479,9 +510,14 @@ def from_combined_dataframe(
 
     @staticmethod
     def from_csv(path_or_buf: Union[pathlib.Path, TextIO]) -> "MultiRegionTimeseriesDataset":
-        return MultiRegionTimeseriesDataset.from_combined_dataframe(
+        mrts = MultiRegionTimeseriesDataset.from_combined_dataframe(
```

Comment:
fwiw I'd prefer `dataset` over `mrts`, since mrts is pretty opaque and I think if you didn't see it created right by `MultiRegionTimeseriesDataset` it's not obvious that its the abbreviation for that

Reply:
Done